src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -149,6 +149,53 @@ public void Method1()
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)]
+        [WorkItem(42986, "https://github.com/dotnet/roslyn/issues/42986")]
+        public async Task TestMethodWithNativeIntegers()
+        {
+            var nativeIntegerAttributeDefinition = @"
```

Comment:
There's a way to have this generate a proper metadata reference:

https://github.com/dotnet/roslyn/blob/cdb15f3c166d2b496e5d845d1c3369d6f36998e5/src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs#L436-L462

That way you can just use the actual feature syntax in the reference, which also means you won't have any issues if this isn't quite expressing reality.